### PR TITLE
added initial version of theme and subtheme code

### PR DESCRIPTION
**Pull request by danpollock**
Hey @mark-walle can you review? 

This is to connect with dashboards/57

### CODE REVIEW

File: cmslite_themes.view.lkml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  }
+
+  dimension: theme {
+    type: string
+    sql: ${TABLE}.theme_id ;;
+  }
+  dimension: theme_id {
+    type: string
+    sql: ${TABLE}.theme_id ;;
+  }
```
Comment (mark-walle) [inline cmslite_themes.view.lkml]: Are these dimensions meant to have the same SQL?

File: cmslite_themes.view.lkml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+  dimension: subtheme {
+    type: string
+    sql: ${TABLE}.subtheme_id ;;
+  }
+  dimension: subtheme_id {
+    type: string
+    sql: ${TABLE}.subtheme_id ;;
+  }
```
Comment (mark-walle) [inline cmslite_themes.view.lkml]: Likewise; are these dimensions meant to have the same SQL?

**Comment by mark-walle** [comment]
@danpollock case expressions seem to make sense to me. Looks good other than my uncertainty on the above dimensions having the same sql

File: cmslite_themes.view.lkml (hunk 2 above)
Comment (danpollock) [reply cmslite_themes.view.lkml]: Yes. 

See the comment above "Will change node_id to title when available". 

Next sprint, when we have the title fields in the metadata, we'll change one of them to the string value.

**Comment by mark-walle** [comment]
Yes I see that, thank you.

**Pull request merged**